content/en/tracing/trace_pipeline/trace_retention.md

```diff
@@ -18,35 +18,35 @@ further_reading:
 
 {{< img src="tracing/apm_lifecycle/retention_filters.png" style="width:100%; background:none; border:none; box-shadow:none;" alt="Retention filters" >}}
 
-With APM, [the ingestion and the retention of traces for 15 days][1] are fully customizable.
+With Datadog APM, [the ingestion and the retention of traces for 15 days][1] are fully customizable.
 
 To track or monitor your volume of ingested and indexed data, see the [Usage Metrics][2] documentation.
 
 ## Retention filters
 
-After spans have been ingested by Datadog, some are kept for 15 days according to the retention filters that have been set on your account.
+After spans have been ingested, some are kept for 15 to 30 days according to the retention filters that have been set on your account.
```

Comment:
this sentence will make people think they can configure the period of retention in their retention filters